app-drupal/index.js

```diff
@@ -19,9 +19,10 @@ _.forEach([1, 2, 3, 4], (num) => {
 });
 
 Drupal.behaviors.designSystem = {
-  attach(context, settings) {
-    designSystem.forEach((component) => {
-      component.enable(context, settings);
+  attach($context, settings) {
+    _.forEach(designSystem, (component) => {
+      console.log(component.name);
```

Comment:
Is name enough context to make sense of the logging?

Reply:
Yep, this is just not-production clarity for new folks coming in to see how this works.